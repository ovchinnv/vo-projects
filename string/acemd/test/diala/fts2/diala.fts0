#
# input file for Struna Plugin via ACEMD
#

ftsm_init {
 defer_init_dyna=yes ! runs initialization stack at first step of dynamics, when coordinates are available
!=======================
 : init
!======================= atom selection :
 : set orient select atomid 1 5 6 7 9 11 15 16 17 19 end
 : set rmsd   select atomid 1 5 6 7 9 11 15 16 17 19 end
!========================
 : list                      ! list atoms
!========================
 : set mass on      ! turn on mass weighting in RMSD computation and orientation
 : set kpar 50      ! force constant for hyperplanes restraints
 : set dpar 0 ! location of hyperplane (different for internal vs. endpoints)
 : set kprp 50      ! default is 0 (no perpendicular restraint)
 : set dprp 2.0     ! default is 1 (unscaled)
 : set proj on      ! forces restrain distance to hyperplane, but not position in the hyperplane (this is needed to compute 1D FE)
! read string coords
! : write cor name string0.cor col center

! write string coords
! : write cor name string0.cor col center
}

ftsm_done {
}

