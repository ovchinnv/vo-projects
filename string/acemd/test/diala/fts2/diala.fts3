#
# input file for Struna Plugin via ACEMD
#

output {
 minwarnlev=-1
}

ftsm_init {
 defer_init_dyna=yes ! runs initialization stack at first step of dynamics, when coordinates are available
!=======================
 : init
!======================= atom selection :
 : set orient select atomid 1 5 6 7 9 11 15 16 17 19 end
 : set rmsd   select atomid 1 5 6 7 9 11 15 16 17 19 end
!========================
 : list                      ! list atoms
!========================
 : set mass on      ! turn on mass weighting in RMSD computation and orientation
 : set kpar 50      ! force constant for hyperplanes restraints
 : set dpar 0.5 ! location of hyperplane (different for internal vs. endpoints)
 : set kprp 50      ! default is 0 (no perpendicular restraint)
 : set dprp 2.0     ! default is 1 (unscaled)
 : set proj on      ! forces restrain distance to hyperplane, but not position in the hyperplane (this is needed to compute 1D FE)
! read string coords
 : read txt name diala22_fts_ini.txt from 2  col left
 : read txt name diala22_fts_ini.txt from 3   col center
 : read txt name diala22_fts_ini.txt from 4 col right
 : copy main ref ! not sure that this is needed, since we are not comparing (or changing) string coords

! write string coords
 : write txt name fts3.txt col center
! : write txt name fts2.txt col left
! : write txt name fts4.txt col right

! statistics output
 : stat forc fcnm data/force3.dat - ! output forces
        dist dnam data/proj3.dat    ! projection variables
 : dyna -
    rstr -           ! restrained dynamics 
    stat staf 1000   ! statistics output

}

ftsm_done {
}

