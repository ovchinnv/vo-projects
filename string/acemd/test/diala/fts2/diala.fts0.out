 MESSAGE (sm_init_from_acemd): FINITE TEMPERATURE STRING METHOD (FTSM) REQUESTED
 MESSAGE (sm_init_from_acemd): INITIALIZING STRING METHOD PLUGIN WITH 22 ATOMS
 MESSAGE (FILES_OPEN): Opening FORMATTED file "diala.fts0" for reading.
 MESSAGE (PARSE_FILE): Reading input file.
 MESSAGE (PARSE_FILE): ======================> Switching to context "FTSM_INIT".
 MESSAGE (PARSE_FILE): defer_init_dyna <= yes
 MESSAGE (PARSE_FILE): Execution stack += "init"
 MESSAGE (PARSE_FILE): Execution stack += "set orient select atomid 1 5 6 7 9 11 15 16 17 19 end"
 MESSAGE (PARSE_FILE): Execution stack += "set rmsd   select atomid 1 5 6 7 9 11 15 16 17 19 end"
 MESSAGE (PARSE_FILE): Execution stack += "list"
 MESSAGE (PARSE_FILE): Execution stack += "set mass on"
 MESSAGE (PARSE_FILE): Execution stack += "set kpar 50"
 MESSAGE (PARSE_FILE): Execution stack += "set dpar 0"
 MESSAGE (PARSE_FILE): Execution stack += "set kprp 50"
 MESSAGE (PARSE_FILE): Execution stack += "set dprp 2.0"
 MESSAGE (PARSE_FILE): Execution stack += "set proj on"
 MESSAGE (PARSE_FILE): ======================> Returning to main context.
 MESSAGE (PARSE_FILE): ======================> Switching to context "FTSM_DONE".
 MESSAGE (PARSE_FILE): ======================> Returning to main context.
 MESSAGE (PARSE_FILE): Input file read.
 MESSAGE (FILES_CLOSE): Closing unit 101
 MESSAGE (sm_init_from_acemd): WILL DEFER EXECUTION OF INITIALIZATION STACK UNTIL DYNAMICS
 MESSAGE (sm_dyna_from_acemd): EXECUTING INITIALIZATION STACK
 MESSAGE (FTSM_EXEC): Executing commands within context "FTSM_INIT"
 MESSAGE (FTSM_EXEC):  =====> "init"
FTSM USING FTSM IN CENTER-OF-MASS COORDINATES. (M-TENSOR/JACOBIAN CALCULATIONS ARE NOT IMPLEMENTED).
FTSM_INIT FOUND     1 REPLICAS.
FTSM_INIT FTSM VERSION 2 WILL BE USED TO SAMPLE HYPERPLANES.
 MESSAGE (FTSM_EXEC):  =====> "set orient select atomid 1 5 6 7 9 11 15 16 17 19 end"
FTSM_ADD_ATOMIC_SET WILL ORIENT STRUCTURES BASED ON    10 ATOMS
FTSM_ADD_ATOMIC_SET ORIENTATION WEIGHTS UNIFORM.
FTSM_ADD_ATOMIC_SET ORIENTATION AND FORCING ATOMS ARE DIFFERENT
 MESSAGE (FTSM_EXEC):  =====> "set rmsd   select atomid 1 5 6 7 9 11 15 16 17 19 end"
FTSM_ADD_ATOMIC_SET WILL APPLY FORCES TO    10 ATOMS
FTSM_ADD_ATOMIC_SET FORCING WEIGHTS UNIFORM.
FTSM_ADD_ATOMIC_SET ORIENTATION AND FORCING ATOMS ARE IDENTICAL
 MESSAGE (FTSM_EXEC):  =====> "list"
 MESSAGE (FTSM_LIST_ATOMS): MINIMAL SYSTEM DOES NOT REPORT TOPOLOGY.
	ORIENTATION ATOMS:
FTSM_LIST_ATOMS COM GROUP #        1:
	       1       1 UNKN    UNKN    UNKN    UNKN
FTSM_LIST_ATOMS COM GROUP #        2:
	       2       5 UNKN    UNKN    UNKN    UNKN
FTSM_LIST_ATOMS COM GROUP #        3:
	       3       6 UNKN    UNKN    UNKN    UNKN
FTSM_LIST_ATOMS COM GROUP #        4:
	       4       7 UNKN    UNKN    UNKN    UNKN
FTSM_LIST_ATOMS COM GROUP #        5:
	       5       9 UNKN    UNKN    UNKN    UNKN
FTSM_LIST_ATOMS COM GROUP #        6:
	       6      11 UNKN    UNKN    UNKN    UNKN
FTSM_LIST_ATOMS COM GROUP #        7:
	       7      15 UNKN    UNKN    UNKN    UNKN
FTSM_LIST_ATOMS COM GROUP #        8:
	       8      16 UNKN    UNKN    UNKN    UNKN
FTSM_LIST_ATOMS COM GROUP #        9:
	       9      17 UNKN    UNKN    UNKN    UNKN
FTSM_LIST_ATOMS COM GROUP #       10:
	      10      19 UNKN    UNKN    UNKN    UNKN
	FORCING AND ORIENTATION ATOMS ARE THE SAME
 MESSAGE (FTSM_EXEC):  =====> "set mass on"
FTSM WEIGHTS SET FROM ATOM MASSES.
 MESSAGE (FTSM_EXEC):  =====> "set kpar 50"
FTSM SETTING PARALLEL RESTRAINT FORCE CONSTANT TO    50.00000
FTSM( LENGTH SCALE ASSUMED TO BE IN UNITS OF THE INTER-REPLICA DISTANCE.)
 MESSAGE (FTSM_EXEC):  =====> "set dpar 0"
 NONFATAL WARNING (FTSM): SETTING DPAR SHOULD NOT NORMALLY BE NEEDED. YOU HAVE BEEN WARNED.
 NONFATAL WARNING (FTSM): DPAR IS EXPRESSED IN UNITS OF 2 x INTER-REPLICA DISTANCE.
FTSM REPLICA NUMBER INVALID OR UNSPECIFIED.
FTSM WILL SET REFERENCE VALUE FOR PARALLEL RESTRAINT   TO   0.000 ON ALL REPLICAS.
 MESSAGE (FTSM_EXEC):  =====> "set kprp 50"
FTSM SETTING PERPENDICULAR RESTRAINT FORCE CONSTANT TO    50.00000
FTSM( LENGTH SCALE ASSUMED TO BE IN UNITS OF THE INTER-REPLICA DISTANCE.)
 MESSAGE (FTSM_EXEC):  =====> "set dprp 2.0"
FTSM REPLICA NUMBER INVALID OR UNSPECIFIED.
FTSM WILL SET OFFSET DISTANCE FOR PERPENDICULAR RESTRAINT TO  0.200E+01 ON ALL REPLICAS.
FTSM( LENGTH SCALE ASSUMED TO BE IN UNITS OF THE INTER-REPLICA DISTANCE.)
 MESSAGE (FTSM_EXEC):  =====> "set proj on"
FTSM WILL RESTRAIN SYSTEM TO PLANE PERPENDICULAR TO PATH.
 NONFATAL WARNING (FTSM): PROJECTION MAY BE ILL-DEFINED BECAUSE ONLY ONE REPLICA IS PRESENT (YOU HAVE BEEN WARNED).
FTSM_DONE CLEANING UP.
